scripts/unit-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Not a fan of this script, documenting env variables and how to use a rakefile is sufficient.